fast64_internal/oot/exporter/export.py

```diff
@@ -103,28 +105,16 @@ def ootPreprendSceneIncludes(outScene: OOTScene, fileData: CData):
 
 def exportScene(
     inSceneObj: Object,
-    transformMatrix: Matrix,
-    f3dType: str,
-    isHWv1: bool,  # is hardware v1
+    outScene: OOTScene,
     sceneName: str,
-    dlFormat: DLFormat,
     savePNG: bool,
     exportInfo: ExportInfo,
     bootToSceneOptions: OOTBootupSceneOptions,
 ):
-    # circular import fix
-    from .scene_folder import modifySceneFiles
-    from .spec import modifySegmentDefinition
-    from ..oot_level_writer import ootConvertScene
-
     checkObjectReference(inSceneObj, "Scene object")
```

Comment:
`checkObjectReference` also should be moved out of this function, as `exportScene` shouldn't need to take in any blender objects as arguments.